### DISCUSSION TITLE
Secondary structure prediction

### DISCUSSION THREAD

**Discussion by Jiacheng06** [facebookresearch/esm#222] [Q&A]
Hi,

Thanks for the great work! I was wondering about how to use ESM to do Q8 secondary structure prediction? You did the experiment in the paper but I cannot find it in the sample notebooks. Is there any sample code for that?

Thanks a lot!

**Comment by tomsercu** [comment]
The code for downstream tasks like secondary structure prediction is not in the `esm` repo, but pretty straightforward to implement on top of the model embeddings. We used the netsurf2 data available here (tab Datasets): https://services.healthtech.dtu.dk/service.php?NetSurfP-2.0
